pkg/manifests/yaml/rte/securitycontextconstrain.yaml

```diff
@@ -0,0 +1,16 @@
+# TODO: check if we can reduce scc capabilities
```

Comment:
please rename to `securitycontextconstraint.yaml` (note the last "t" before `.yaml`)

Reply:
done